charts/dapr/charts/dapr_rbac/templates/dashboard.yaml

```diff
@@ -7,7 +7,11 @@ metadata:
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
 ---
+{{- if eq .Values.global.rbac.namespaced true }}
```

Comment:
I think

```
{{- if .Values.global.rbac.namespaced }}
```

would be enough?

Reply:
I've encountered Helm oddities before, not taking my chances :)